### PR TITLE
Eliminate redundant "for" in trigger description

### PR DESCRIPTION
**Pull request by krazos**
## Proposed change

For automation triggers with a duration parameter, the trigger description includes the word "for" in both the "duration" variable and again when the "duration" variable is appended to the "base" string, resulting in the word "for" appearing twice in the trigger description.

Example:  "When Location Mode changes from Home to Away for for 00:30:00"

I propose to fix the redundancy by removing the word "for" from the "duration" variable.

## Type of change

- [ ] Dependency upgrade
- [X] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

N/A

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

**Comment by krazos** [comment]
Screenshot from 2022-09-08 19-23-28

File: src/data/automation_i18n.ts [hunk 1]
```diff
@@ -143,11 +143,11 @@ export const describeTrigger = (
...
       let duration: string;
       if (typeof trigger.for === "number") {
-        duration = `for ${secondsToDuration(trigger.for)!}`;
+        duration = `${secondsToDuration(trigger.for)!}`;
       } else if (typeof trigger.for === "string") {
-        duration = `for ${trigger.for}`;
+        duration = `${trigger.for}`;
       } else {
-        duration = `for ${JSON.stringify(trigger.for)}`;
+        duration = `${JSON.stringify(trigger.for)}`;
```
Comment (bramkragten) [inline src/data/automation_i18n.ts:150]:
```suggestion
        duration = `${formatDuration(trigger.for)}`;
```

```ts
import { formatDuration } from "../common/datetime/format_duration";
```

**Review by bramkragten — APPROVED** [review]

**Comment by krazos** [comment]
Thank you. I was planning to separately raise the issue of converting the JSON presentation of hours/minutes/seconds to plain language, but I wasn't quite sure how to fix it myself.

**Comment by krazos** [comment]
While folks are thinking about the automation trigger/condition/action descriptions, here is a suggestion re: use of frontend state name with localization rather than actual (backend) state:

https://github.com/home-assistant/frontend/discussions/13674

**Pull request merged**